packages/test/src/ExperimentCrudUtils.ts

```diff
@@ -0,0 +1,171 @@
+import { AuditBehaviorTypes, Query } from '@labkey/api';
+import { IntegrationTestServer, RequestOptions, successfulResponse } from './integrationUtils';
+import { sleep } from './utils';
+
+export async function insertRows(server: IntegrationTestServer, rows: any[], schema: string, dataType: string, folderOptions: RequestOptions, userOptions: RequestOptions, auditBehavior: AuditBehaviorTypes = AuditBehaviorTypes.DETAILED, debug?: boolean) : Promise<any[]> {
+    const response = await server.post('query', 'insertRows', {
+        schemaName: schema,
+        queryName: dataType,
```

Comment:
minor: I would probably just call this parameter `queryName` instead of `dataType` to be consistent